.github/workflows/server-benchmark.yml

```diff
@@ -0,0 +1,35 @@
+name: Server SDK tests and benchmarks
+# This job will run the server SDK integration tests amd benchmarks using the Pokémon service model.
+on:
+  pull_request:
+    types:
+    - opened
+    - reopened
+    - synchronize
+env:
+  java_version: 11
+  rust_version: 1.56.1
```

Comment:
It looks like you're not installing the Rust toolchain in the job, so it's going to use whatever is installed by default on the image rather than this version.

Reply:
Ah, thanks for pointing this out. I'll fix.